### ISSUE TITLE
brew reinstall at-spi2-atk -s from sources fails with  "Native dependency 'atspi-2' not found Why! 

### ISSUE THREAD

**Issue by JuPlutonic** [Linuxbrew/homebrew-core#7661]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x ] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [x ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/caskroom/homebrew-cask/issues/new. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
**brew reinstall at-spi2-atk -s**
- [x] ran `brew update` and can still reproduce the problem?
Yes
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
```
Warning: You have unlinked kegs in your Cellar
Leaving kegs unlinked can lead to build-trouble and cause brews that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  gtk+
  rbenv
```
Yes :(
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?
[here](https://gist.github.com/8077cbb95f2d6bd78f1d28db80ecc76b)

#7175 after atk.pc found got `Package 'atspi-2', required by 'atk-bridge-2.0', not found but **at-spi2-core** and **at-spi2-atk** presented

+ Strange:

brew linkage --test at-spi2-atk 
Unwanted system libraries:
>>>  libatk-1.0.so.0
  libatspi.so.0

brew linkage --test at-spi2-core
Unwanted system libraries:
>>>  libX11.so.6
  libXtst.so.6
  libatspi.so.0
  libdbus-1.so.3
  libgio-2.0.so.0
  libglib-2.0.so.0
  libgobject-2.0.so.0

**Comment by JuPlutonic** [comment]
After I copied 
```
cp -R cp -R `brew --repository`/Cellar/at-spi2-core/2.28.0/lib/x86_64-linux-gnu/* cp -R `brew --repository`/Cellar/at-spi2-core/2.28.0/lib
```
I finally got **at-spi2-core** reinstalled from sources.

**Comment by iMichka** [comment]
I'll open a PR to fix this, using the same strategy as in #7663

**Comment by iMichka** [comment]
See #7686

**Comment by iMichka** [comment]
Should be fixed by #7686
